js/src/components/app-input-control/index.scss

```diff
@@ -10,4 +10,10 @@
 			white-space: normal !important;
 		}
 	}
+
+	// Hack to prevent the read-only input from having the focused border style.
+	// Ref: https://github.com/WordPress/gutenberg/blob/%40wordpress/components%4012.0.8/packages/components/src/input-control/styles/input-control-styles.js#L238-L247
+	&--no-pointer-events {
+		pointer-events: none;
+	}
```

Comment:
❔ Since it is a "Hack to prevent the _read-only input_ from having the focused border style", I'm thinking it may be better to name the class like `&--read-only`? 

Side note: I don't think [`:read-only`](https://developer.mozilla.org/en-US/docs/Web/CSS/:read-only) pseudo-class would work here (since the CSS class applies to div), right?

Reply:
This class is exactly applying `pointer-events: none;` style. Not sure why it's better to name with `&--read-only`?

I rephrased the code comments in https://github.com/woocommerce/google-listings-and-ads/pull/1268/commits/e56653d518494591231d47562f7e1cc18b43566b. Hope the intention could be expressed more clearly.

> Side note: I don't think [`:read-only`](https://developer.mozilla.org/en-US/docs/Web/CSS/:read-only) pseudo-class would work here (since the CSS class applies to div), right?

Right, a read-only DOM is still focusable.

Reply:
> Not sure why it's better to name with `&--read-only`?

The reason is to make the CSS class describe what it is, instead of how it is supposed to look like. For example, if we want a read-only element to have no pointer events, display text in yellow color and with red background, the class naming "read-only" would be better than "no-pointer-events".